dispatcher.go

```diff
@@ -66,14 +68,29 @@ func splitAddress(addr string) (mailbox, domain string, err error) {
 	}
 }
 
+func dispatcherDeliveryLogger(l log.Logger, msgMeta *module.MsgMetadata) *log.Logger {
```

Comment:
Rebase the your commit branch on upstream master and use the fixed deliveryLogger function.

Here are Git magic to do so, make sure your working directory is clean (you have no uncommited changes):
```
git remote add upstream https://github.com/foxcpp/maddy.git
git fetch upstream
git rebase upstream/master
```